cmd/cmd.go

```diff
@@ -0,0 +1,176 @@
+// Package cmd - package
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github/Asac2142/go-cli-app/file"
+	"github/Asac2142/go-cli-app/task"
+)
+
+const (
+	add            string = "add"
+	update         string = "update"
+	deleteTask     string = "delete"
+	markInProgress string = "mark-in-progress"
+	markDone       string = "mark-done"
+	list           string = "list"
+)
+
+// HandleTrackerCLI - Handles user's input
+func HandleTrackerCLI(f *file.File[task.TContent]) {
+	task := task.NewTask(f)
+	args := os.Args[1:]
+
+	if len(args) == 0 {
+		err := errors.New("Invalid input")
+		printError(err)
+	}
+
+	switch args[0] {
+	case add:
+		err := addTask(&args, task)
+		printError(err)
+	case update:
+		err := updateTask(&args, task)
+		printError(err)
+	case deleteTask:
+		err := removeTask(&args, task)
+		printError(err)
+	case markInProgress:
+		err := updateStatus(&args, markInProgress, task)
+		printError(err)
+	case markDone:
+		err := updateStatus(&args, markDone, task)
+		printError(err)
+	case list:
+		err := listTasks(&args, task)
+		printError(err)
+	default:
+		err := errors.New("Invalid task command")
```

Comment:
considering you'll wrap this error after the switch with `args[0]` you can use error text "command not found" or something similar.

Another node, all the errors and wrap contexts must start with lowercase letter and must not contain words like "error", "failed", etc, because in the end you'll have a single sentence like (I'm just making it up)
```
list: load tasks: read file: file not found
```